### PR TITLE
Add Sentry txn observability for webhooks

### PR DESCRIPTION
**Pull request by brian-lou**
This PR adds Sentry txn tracing to the webhook endpoints

### CODE REVIEW

File: src/webhooks/index.ts [hunk 1]
```diff
@@ -9,38 +11,47 @@ import { kafkactlWebhook } from './kafka-control-plane/kafka-control-plane';
...
+) => Promise<void>;
+
 // Error handling wrapper function
 export async function handleRoute(
-  handler,
+  handler: WebhookHandler,
   request: FastifyRequest,
-  reply: FastifyReply
+  reply: FastifyReply,
+  name: string
```
Comment (IanWoodard) [inline src/webhooks/index.ts:24]: Something about this makes me wish there was a better way, but this makes sense and works

File: src/webhooks/index.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import '@sentry/tracing';
```
Comment (IanWoodard) [inline src/webhooks/index.ts:1]: Any idea why this is needed? Is it needed? I know we had this previously, but I am not sure it is needed.

**Review by IanWoodard — APPROVED** [review]
Left a few comments, but otherwise lgtm

File: src/webhooks/index.ts (hunk 2 above)
Comment (brian-lou) [reply src/webhooks/index.ts:1]: In order to initialize Sentry this has to be called

**Pull request merged**